internal/env/env.go

```diff
@@ -139,3 +140,32 @@ func StringSlice(key string, validator func(string) error) ([]string, error) {
 	}
 	return ret, nil
 }
+
+func Duration(key string) (time.Duration, error) {
```

Comment:
The `validator` argument is missing from this func, as well as its execution.

Reply:
ah, in hurry I missed it :(